Dockerfile.datab

```diff
@@ -0,0 +1,7 @@
+FROM mysql:5.7
```

Comment:
Tiny nit: "Dockerfile.database" or "Dockerfile.db" would be a better name for this file.